generate-css.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cd "$(dirname "$0")"
+
+(cd sandworm && eval $(opam env) && dune build main.css)
```

Comment:
I believe it would be clearer to do `opam exec` instead of modifying the environment only to launch the binary in the right environment.